CV Backend/src/main/java/com/qa/demo/Interoperability/AccountController.java

```diff
@@ -0,0 +1,233 @@
+package com.qa.demo.Interoperability;
+
+
+
+import com.qa.demo.domain.Account;
+import com.qa.demo.domain.PasswordObject;
+import com.qa.demo.repository.AccountRepository;
+import com.qa.demo.service.MongoUserDetailsService;
+import org.bson.BsonBinarySubType;
+import org.bson.types.Binary;
+import org.bson.types.ObjectId;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.data.mongodb.core.query.Query;
+import org.springframework.data.mongodb.core.query.Update;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PostAuthorize;
+import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.validation.Valid;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.security.Principal;
+import java.util.Base64;
+import java.util.List;
+
+import static org.springframework.data.mongodb.core.query.Criteria.where;
+
+@Transactional
+@RestController
+@RequestMapping("/account")
+@EnableGlobalMethodSecurity(prePostEnabled = true)
+public class AccountController {
```

Comment:
the same issues persist in this whole class, fix all of them based on the comments below